external-builds/pytorch/patches/pytorch/v2.7.0/pytorch/hipified/0012-Patch-ifdef-for-__builtin_clz-on-Windows.patch

```diff
@@ -0,0 +1,26 @@
+From 888d52302e9c6514ac3aadebc56d27a460e64503 Mon Sep 17 00:00:00 2001
+From: Aaryaman Vasishta <jem456.vasishta@gmail.com>
+Date: Sun, 4 May 2025 22:24:50 +0530
+Subject: [PATCH 12/14] Patch ifdef for `__builtin_clz` on Windows.
+
+Co-authored-by: Aaryaman Vasishta <jem456.vasishta@gmail.com>
```

Comment:
nitpick: this patch already has me as author, so this co-authored-by is probably not needed.  But if it's harmless then no need to touch it

Reply:
Ah, over-eager copy/paste. Nice catch, though I think It should be harmless.

Reply:
Dropped the redundant co-authored by while rebasing again.